contracts/src/v0.8/functions/dev/1_0_0/FunctionsCoordinator.sol

```diff
@@ -230,16 +226,8 @@ contract FunctionsCoordinator is OCR2Base, IFunctionsCoordinator, FunctionsBilli
         _fulfillAndBill(requestIds[i], results[i], errors[i], onchainMetadata[i], offchainMetadata[i])
       );
 
-      if (result == FulfillResult.USER_SUCCESS || result == FulfillResult.USER_ERROR) {
+      if (result == FulfillResult.USER_SUCCESS || result == FulfillResult.USER_ERROR || result == FulfillResult.SUBSCRIPTION_BALANCE_INVARIANT_VIOLATION) {
         emit OracleResponse(requestIds[i], msg.sender);
-      } else if (result == FulfillResult.INVALID_REQUEST_ID) {
```

Comment:
We could keep those if we want.

Reply:
I'm for removing if we don't plan to use them. Makes it a lot cleaner